geocity/apps/forms/models.py

```diff
@@ -518,6 +520,14 @@ class Form(models.Model):
         related_name="map_widget_configuration_form",
         verbose_name=_("Configuration de la carte avancée"),
     )
+    quick_access_slug = models.TextField(
+        blank=True,
+        default=uuid.uuid4,
+        unique=True,
+        validators=[validate_slug],
+        verbose_name=_("URL courte"),
+    )
```

Comment:
Could you please add (after rename `q` to `form` of course):
`help_text="Permettant d'accéder directement au formulaire par l'url: https://geocity.ch/?form=demande-macaron",`

Reply:
Just add the ":" before the example to be consitant with the other help texts.

And should be also in the migration?

Reply:
Yep, sorry for the oversight :+1: